changes to allow maketarget work with makeclothes

black-punkduck opened the pull request:
Scale and basemesh are needed from external sources. If a standard
basemesh is loaded, scale should be 1.0. So if the attribute isn't there, 1.0 is assumed.

Furthermore I changed output to the one makehuman uses for the standard
targets. Now it is possible to load an existing
standard target, change it and save it, the difference will only be the
new lines.

black-punkduck committed:
changes to allow maketarget work with makeclothes

Scale and basemesh are needed from external sources. If a standard
basemesh is loaded, scale should be 1.0. So if the attribute isn't there, 1.0 is assumed.

Furthermore I changed output to the one makehuman uses for the standard
targets. Now it is possible to load an existing
standard target, change it and save it, the difference will only be the
new lines.

joepal1976 commented on maketarget/loadprimarytarget.py:
```diff
@@ -28,14 +28,15 @@ def execute(self, context):
         sks = obj.data.shape_keys
         pt = sks.key_blocks["PrimaryTarget"]
```
(Comment written just to demonstrate what happens)
Maybe we should use better variable names.

joepal1976 reviewed (approved):
Looks good

black-punkduck committed:
changes to work with flexible target name

I added a box, where the user can enter the primary target name. This
name will be used as a filename (replacing all non-alpha and non-number
characters to "-"). If a target is loaded, filename will be used as a
target (but linked to object, not to scene). I changed the code to
produce lines a bit, unfortunately I created "0 0 0" lines again.
